web-console/future-openshift/switcher-samples/switcher-samples.md

```diff
@@ -0,0 +1,53 @@
+# YAML Switcher and Samples
+
+## Overview
+* If a resource has a creation form (and YAML), the form will load on default.
+  * After the resource is created, users can edit the resource in YAML only.
+  * In the future, inline edit will be investigated as a way to allow users to edit fields in a form-like manner.
+* Samples available in YAML editor will show YAML code to be copied and pasted as well as a "Use this sample" button that will overwrite the YAML with the sample.
+* Samples available in form will show a "Use this sample" button and will fill in form fields.
+* Changes made in YAML will be reflected when switching to form view, and vice versa.
+  * An inline notification should appear on the form view notifying users that any additional fields they add in YAML will be saved.
+
+
+## Switcher
+![1](img/switcher-form.png)
+
+![2](img/switcher-YAML.png)
+
+![3](img/switcher-form2.png)
+
+* When a resource has a creation form, users can switch back and forth between the YAML editor and the form using the toggle in the upper right hand corner.
+* Edits made in YAML will be reflected in form view and vice versa.
+* Users can add extra fields in YAML that do not necessarily have a related form field.
+  * An inline notification will be shown on the form view in these cases to inform users that their changes or additions to YAML will be saved.
+
+
+## Samples
+![4](img/sample1.png)
+
+![5](img/sample2.png)
+
+![6](img/no-sample.png)
+
+* YAML editors with samples will be loaded with the samples panel open.
```

Comment:
The "x" doesn't feel quite right to me given having the panel open is the default state. Unlike other places that we use an overlay panel, the user didn't do something to open this one. I wonder if there should be an explicit "hide samples" action instead of/in addition to the "x".